### PR TITLE
feat: Add info fields to GQL image schema

### PR DESCRIPTION
**Pull request by fregataa**
### GQL query example
```graphql
query ImageList {
    images {
        id
        name
        namespace
        base_image_name
        tags {
            key
            value
        }
        version
        registry
        architecture
    }
}
```

**Checklist:** (if applicable)

- [x] Milestone metadata specifying the target backport version
- [ ] API server-client counterparts (e.g., manager API -> client SDK)

### CODE REVIEW

**Comment by fregataa** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/lablup/backend.ai/2939?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2939** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai/2939?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2938** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai/2938?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fregataa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by agatha197 — APPROVED** [review]
LGTM

**Review by agatha197 — CHANGES_REQUESTED** [review]
If tags include `customized_` in the key, could you change the tag value to the value of the label whose key is `ai.backend.customized-image.name` ??



Currently, it looks like this. Users will identify images based on the content they input in the session name when committing the session. The images are stored with hash values. Therefore, it is difficult to differentiate which image was committed with what name using the current tag.

**Review by agatha197 — APPROVED** [review]

**Review by agatha197 — APPROVED** [review]

**Review by agatha197 — APPROVED** [review]

**Review by agatha197 — CHANGES_REQUESTED** [review]
Please change target version and codes to 24.12.0

File: src/ai/backend/manager/models/gql_models/image.py [hunk 1]
```diff
@@ -314,10 +324,14 @@ class Meta:
         interfaces = (AsyncNode,)
 
     row_id = graphene.UUID(description="Added in 24.03.4. The undecoded id value stored in DB.")
-    name = graphene.String()
+    name = graphene.String(deprecation_reason="Deprecated since 24.12.0. use `namespace` instead")
+    namespace = graphene.String(description="Added in 24.12.0..")
+    base_image_name = graphene.String(description="Added in 24.12.0..")
```
Comment (agatha197) [inline src/ai/backend/manager/models/gql_models/image.py]:
```suggestion
    base_image_name = graphene.String(description="Added in 24.12.0.")
```
And it's the same below.

**Review by agatha197 — CHANGES_REQUESTED** [review]

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**